app/page.tsx

```diff
@@ -1,14 +1,15 @@
+"use client";
 import Hero from "@/components/Hero";
 import Banners from "@/components/Banners";
-import PartnerBenefits from "@/components/PartnerBenefits";
+import CalendariumBenefits from "@/components/CalendariumBenefits";
 import NewsAndEvents from "@/components/NewsAndEvents";
 
 export default function Home() {
   return (
     <main className="bg-background">
       <Hero />
       <Banners />
-      <PartnerBenefits />
+      <CalendariumBenefits />
```

Comment:
We should rethink of a section to put here. I know this is only a POC tho 💪🏼

Reply:
Maybe we put this in commentary, and see later what to do with this section